docs/dev_guide/command-line-options-reference.rst

```diff
@@ -38,6 +38,9 @@ in alphabetical order.
    * - ``=c_cxx_standard_library``
      - A set of functions from the C and C++ standard libraries that are allowed
        to be used in SYCL device code.
+   * - ``--change-filename-extension=<file>``
+     - DEPRECATED: Specifies which extension(s) of migrated file(s) will be changed,

```

Comment:
"DEPRECATED: A comma-separated list that specifies which extensions of migrated files will be changed. Default..."